src/main/java/uk/gov/hmcts/darts/cases/helper/AdvancedSearchRequestHelper.java

```diff
@@ -138,6 +148,18 @@ private List<Predicate> addCourthouseCriteria(GetCasesSearchRequest request, Cri
         return predicateList;
     }
 
+    private List<Predicate> addUserSecurityRolesCriteria(GetCasesSearchRequest request, CriteriaBuilder criteriaBuilder, Root<CourtCaseEntity> caseRoot) {
+
+        List<Predicate> predicateList = new ArrayList<>();
+        Join<CourtCaseEntity, UserAccountEntity> userJoin = joinUser(caseRoot);
+        predicateList.add(criteriaBuilder.equal(
+                              userJoin.get(UserAccountEntity_.EMAIL_ADDRESS),
```

Comment:
Think this criteria will need to use equal lower both sides like e.g.
https://github.com/hmcts/darts-api/blob/a595b7cb11d92dfe7bdc64293d749c7dc359d0a8/src/main/java/uk/gov/hmcts/darts/authorisation/service/impl/AuthorisationServiceImpl.java#L84

Reply:
Changed criteriaBuilder to lower case